examples/statistics/errorbar_demo_features.py

```diff
@@ -1,37 +1,44 @@
 """
-Demo of errorbar function with different ways of specifying error bars.
+===================================================
+Demo of the different ways of specifying error bars
+===================================================
 
-Errors can be specified as a constant value (as shown in `errorbar_demo.py`),
-or as demonstrated in this example, they can be specified by an N x 1 or 2 x N,
-where N is the number of data points.
+Errors can be specified as a constant value (as shown in
+`errorbar_demo.py`), or as demonstrated in this example, they can be
+specified by an N x 1 or 2 x N, where N is the number of data points.
```

Comment:
I think a word is missing here: an N x 1 or 2 x N _array_. Maybe even drop the " x 1", because it's a bit confusing that the shape seems to transpose itself. But since N x 1 ~ 1 x N ~ N, just N might be okay?